ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,827 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examine trace from workload execution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot task residency and `sched_util` and `sched_load` for the workload task, along with the expected mean value for util_avg. Note that assuming the system was under little load, so that the task was RUNNING whenever it was RUNNABLE, `load_avg` and `util_avg` should be the same. \n",
+    "\n",
+    "Call `examine_experiment` with different experiment indexes to get plots for runs at different frequencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "signals = ['util_avg', 'load_avg']\n",
+    "def examine_experiment(idx):\n",
+    "    experiment = t.experiments[idx]\n",
+    "    \n",
+    "    [freq] = experiment.conf['cpufreq']['freqs'].values()\n",
+    "    print \"Experiment ran at frequency {}\".format(freq)\n",
+    "    events = t.te.test_conf[\"ftrace\"][\"events\"]\n",
+    "    \n",
+    "    # todo add get_trace method\n",
+    "    tasks = experiment.wload.tasks.keys()\n",
+    "    #trace = Trace(t.te.platform, experiment.out_dir, events, tasks)\n",
+    "    print 'Trace plot:'\n",
+    "    plot_trace(t.get_trace(experiment).ftrace)\n",
+    "    \n",
+    "    # Get observed signal\n",
+    "    signal_df = t.get_sched_task_signals(experiment, signals)\n",
+    "    # Get expected average value for util_avg signal\n",
+    "    expected_util_avg_mean = t.get_expected_util_avg(experiment)\n",
+    "    \n",
+    "    # Plot task util_avg signal with expected mean value\n",
+    "    util_avg_mean = pd.Series([expected_util_avg_mean], name='expected_util_avg', index=[signal_df.index[0]])\n",
+    "    df = pd.concat([signal_df, util_avg_mean], axis=1).ffill()\n",
+    "    ILinePlot(df, column=signals + ['expected_util_avg'], drawstyle='steps-post',\n",
+    "             title='Scheduler task signals').view()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Experiment 0:    307200Hz\n",
+      "Experiment 1:    614400Hz\n",
+      "Experiment 2:    902400Hz\n",
+      "Experiment 3:   1209600Hz\n",
+      "Experiment 4:   1516800Hz\n",
+      "Experiment 5:   1824000Hz\n",
+      "Experiment 6:   2150400Hz\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i , f in enumerate(freqs):\n",
+    "    print \"Experiment {}:{:10d}Hz\".format(i, f)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Experiment ran at frequency 307200\n",
+      "Trace plot:\n"
```

Comment:
The [solution proposed by Javi](ARM-software/trappy/issues/239)  is actually something we should better document/advertise on the LISA side as well. ;-)

Perhaps we can wrap the required export commands into a new `lisa-export` command?